src/atoms/Typography/Typography.tsx

```diff
@@ -5,6 +5,15 @@ export const Typography = styled.p`
   color: ${props => props.theme.text};
   font-family: Lato, sans-serif;
   line-height: 1.5;
+
+  a {
+    color: ${props => props.theme.link};
```

Comment:
I think we should use the "links in line" colors here and make it bold, thoughts? We could potentially have a different component for standalone links, but then we'd need a dedicated link component again. https://app.zeplin.io/project/5b0334f5e91e8c481645ad56/screen/5bb23e07eff8754297471dff